dart/lib/src/protocol/contexts.dart

```diff
@@ -28,41 +28,49 @@ class Contexts extends MapView<String, dynamic> {
         });
 
   /// Deserializes [Contexts] from JSON [Map].
-  factory Contexts.fromJson(Map<String, dynamic> data) {
+  // ignore: strict_raw_type
+  factory Contexts.fromJson(Map data) {
+    // This class should be deserializable from Map<String, dynamic> and Map<Object?, Object?>,
+    // because it comes from json.decode which is a Map<String, dynamic> and from
+    // methodchannels which is a Map<Object?, Object?>.
+    // Map<String, dynamic> and Map<Object?, Object?> only have
+    // Map<dynamic, dynamic> as common type constraint
```

Comment:
the leys are always a String though, can we force the type here somehow?

Reply:
I've tried various things, but the only way which worked was iterating through each map and casting each individual entry which seems like overkill.